src/components/VaultRow/lib/VaultTable.tsx

```diff
@@ -1,25 +1,35 @@
+import clsx from "clsx"
 import { FC, PropsWithChildren } from "react"
 
 import { Table, TableBody, TableHeader, TableRow } from "@/components/Table"
 import { TxSettingsPopover } from "@/components/TxSettingsPopover"
 
 type VaultTableProps = {
   label: string
+  earningEnabled: boolean
 }
 
 export const VaultTable: FC<PropsWithChildren<VaultTableProps>> = ({
   children,
   label,
+  earningEnabled,
```

Comment:
I was wondering what this prop did as I have reviewed up until I got to this file. I think maybe we ought to name this prop in a more explicit way. Something like `showEarningsColumn` would be much clearer IMO.